tutorials/hana-dbx-hcc/hana-dbx-hcc.md

```diff
@@ -0,0 +1,141 @@
+---
+parser: v2
+auto_validation: true
+time: 10
+tags: [tutorial>beginner, software-product-function>sap-hana-cloud--sap-hana-database, software-product-function>sap-hana-cloud--data-lake]
+primary_tag: software-product>sap-hana-cloud
+---
+
+# Query databases using the SQL Console in SAP HANA Cloud Central
+<!-- description --> Learn how the SQL Console can be used within SAP HANA Cloud Central to quickly query a selected database.  
+
+## Prerequisites
+- An SAP HANA Cloud database
+- You have completed [this](hana-dbx-create-schema) tutorial which creates a database schema for an SAP HANA Cloud, SAP HANA database.
+- You have completed [this](hana-cloud-dl-clients-overview) tutorial which creates a database schema for an SAP HANA Cloud, data lake Relational Engine
+
+## You will learn
+  - How to open a SQL Console, specify the credentials, and set the current schema
+  - An overview of the functionality provided in the SQL Console
+
+---
+
+### Open a SQL Console
```

Comment:
According to documentation (not 100%, I admit), SQL console is lowercase:

![image](https://github.com/sap-tutorials/Tutorials/assets/11659786/566c3d39-c6e3-4e91-be6d-00981806a69c)

Step names are sentence case so it should be lower case here